sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -397,7 +397,7 @@ public async Task StartRecognizeReceiptsCanParseMultipageForm(bool useStream)
                 operation = await client.StartRecognizeReceiptsFromUriAsync(uri, options);
             }
 
-            RecognizedReceiptCollection recognizedReceipts = await operation.WaitForCompletionAsync(PollingInterval);
+            RecognizedFormCollection recognizedReceipts = await operation.WaitForCompletionAsync(PollingInterval);
```

Comment:
nit: would it make sense to call this `recognizedForms` instead?  I'm on the fence.

Reply:
I think your suggestion makes sense. Updated `recognizedReceipts`, `recognizedReceipt` and `receiptIndex` to `recognizedForms`, `recognizedForm` and `formIndex`.